### ISSUE TITLE
Partition level granularity in reporting

### ISSUE THREAD

**Issue by juhoautio** [sundy-li/burrowx#11]
Is it true that burrowx only supports reporting on topic level, not on partition level?

I'm surprised to if that's the case, because at least in my experience many consuming problems are related to uneven split of partitions, or for example single broken/stalled partition. Because of this it would be really helpful to be able to see the consumer offsets & max offsets & latest commit timestamp on partition level.

I took a look at the code and at least `ConsumerFullOffse`t seems to store the `MaxOffset` on topic level only:

https://github.com/sundy-li/burrowx/blob/57c5a482130ee3c75e2ea808da5acaccae3446f3/monitor/metric.go#L32-L43

What's the role of the `partitionMap` btw?

**Comment by sundy-li** [comment]
Yeah, you are right. It makes sense, I would split the offset/lag to each partition in recent days.

**Comment by sundy-li** [comment]
@juhoautio   it's the implementation of partition-level splitting. I already create a pr for that #12
